src/ResourceManager/Sql/Commands.Sql/ImportExport/Cmdlet/ImportExportCmdletBase.cs

```diff
@@ -0,0 +1,111 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System.Security;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.Commands.Common.Authentication.Models;
+using Microsoft.Azure.Commands.Sql.Common;
+using Microsoft.Azure.Commands.Sql.Database.Cmdlet;
+using Microsoft.Azure.Commands.Sql.ImportExport.Model;
+using Microsoft.Azure.Commands.Sql.ImportExport.Service;
+
+namespace Microsoft.Azure.Commands.Sql.ImportExport.Cmdlet
+{
+    public abstract class ImportExportCmdletBase : AzureSqlCmdletBase<AzureSqlDatabaseImportExportBaseModel, ImportExportDatabaseAdapter>
+    {
+        /// <summary>
+        /// Gets or sets the name of the database server to use.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 1,
+            HelpMessage = "SQL Database server name.")]
+        [ValidateNotNullOrEmpty]
+        public string ServerName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the type of the storage key to use.
+        /// </summary>
+        [Parameter(Mandatory = true, HelpMessage = "The type of the storage key")]
+        [ValidateNotNullOrEmpty]
+        public StorageKeyType StorageKeyType
```

Comment:
Why use these values instaed of using StorageContext?  The user can create a StorageContext using the Storage cmdlets